client/my-sites/checklist/wpcom-checklist/checklist-prompt/task.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+class ChecklistPromptTask extends PureComponent {
+	static propTypes = {
+		buttonText: PropTypes.node,
```

Comment:
`PropTypes.node` works here but I'm wondering if it is too general. `PropTypes.string` could be enough? The same goes to `description` and `title`. If it's just me overlooking an intended flexibility, please ignore me :)

Reply:
Good question; this was carried over from the checklist banner. I suspect the descriptions might contain links or other formatting, but I don't see why the button text or title would for our purposes. I'll update those two to be more specific!